src/app/Http/Requests/UpdatePlaceRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Http\Livewire\Forms\Coordinates;
+
+class UpdatePlaceRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+                'name' => 'required|string|max:255|unique:places,name',
```

Comment:
it won't work like this.
Imagine you wanna update the description. Then the name would be the same and would already exist in the DB. You should mention the id to avoid errors.

Reply:
https://laravel.com/docs/10.x/validation#rule-unique

used in event this way :
`"name" => "required|unique:events,name," . $this->route('event')->id . "|string",`